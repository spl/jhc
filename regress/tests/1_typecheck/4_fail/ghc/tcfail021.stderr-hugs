ERROR "tcfail021.hs":8 - Pattern binding illegal in instance declaration
