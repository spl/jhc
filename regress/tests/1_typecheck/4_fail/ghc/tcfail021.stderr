
tcfail021.hs:8:5:
    Pattern bindings (except simple variables) not allowed in instance declarations
	(==, /=) = (\ x -> \ y -> True, \ x -> \ y -> False)
